### ISSUE TITLE
Big-board views got turned into single-column, and names center-aligned

### ISSUE THREAD

**Issue by mileswwatkins** [nprapps/elections18-graphics#139]
<img width="783" alt="screen shot 2018-09-19 at 16 47 03" src="https://user-images.githubusercontent.com/4959135/45780683-19aab180-bc2c-11e8-94de-86c06138f5a2.png">

![image](https://user-images.githubusercontent.com/4959135/45780699-216a5600-bc2c-11e8-855b-0897d349a99f.png)

**Comment by alykat** [comment]
I think some of the accessibility updates stepped on some recent board styling changes. Should be fixed now.
